forge-gui/res/cardsfolder/upcoming/cleopatra_exiled_pharaoh.txt

```diff
@@ -0,0 +1,12 @@
+Name:Cleopatra, Exiled Pharaoh
+ManaCost:2 B G
+Types:Legendary Creature Human Noble
+PT:2/4
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ At the beginning of your end step, put a +1/+1 counter on each of up to two other target legendary creatures.
```

Comment:
also include trigger "flavor names" in description

Reply:
Duly noted. I will say I noticed their inclusion was a bit inconsistent in existing scripts.